precompiles/Modexp.yul

```diff
@@ -119,7 +129,7 @@ object "ModExp" {
             /// @param resPtr The pointer to where you want the result to be stored
             function bigUIntBitOr(lhsPtr, rhsPtr, nLimbs, resPtr) {
                 // +------------+-----------------------+-------------------------------+-------------------------------+-------------------------------+-----------------+-----------------+--------------------------------------+
-                // | Iteration  |       offset_i        |           ptr_lhs_i           |           ptr_rhs_i           |           ptr_res_i           |   value_lhs_i   |   value_rhs_i   |             value_res_i              |
+                // | Iteration  |       currentOffset   |           lhsCurrentPtr       |           rhsCurrentPtr       |           resCurrentPtr       | lhsCurrentValue | rhsCurrentValue |           resCurrentValue            |
```

Comment:
This seems like a good step by step example. However, I'm not sure if we should keep it as a comment within the code, maybe we can remove it as part of the cleanup

Reply:
I agree, we can have it in `modexp` docs.

Reply:
Moved the comment to the API Docs [fa0c59f](https://github.com/lambdaclass/zksync_era_precompiles/pull/156/commits/fa0c59fdbf1f845fab2c49621582f2528b0924a0).